machine_configuration/README.md

```diff
@@ -87,6 +87,23 @@ $ lsmod | grep ast
 ```
 Ensure that ast driver is not loaded.
 
+# Machine Configuration for Provisioning Intel QAT
+
```

Comment:
Let's move this to MCO. And I think we should remove the QAT machine config pool for simplicity purposes.